tests/components/neptune_apex/test_config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Test the Neptune Apex config flow."""
+from asynctest import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.neptune_apex.config_flow import CannotConnect, InvalidAuth
+from homeassistant.components.neptune_apex.const import DOMAIN
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.neptune_apex.config_flow.ApexHub.authenticate",
```

Comment:
We want to patch the library not our code.
```suggestion
        "homeassistant.components.neptune_apex.config_flow.Apex.validate_connection",
```